### PR TITLE
Add a join slack popover to the home page

### LINKED ISSUE #201: Create Slack Channel notification
New site should have a Join Slack notification thingy. Less intrusive than what's there today.

Maybe a toast? https://getbootstrap.com/docs/5.2/components/toasts/

### PR DESCRIPTION
**Pull request by plumbis**
This creates a static popover on the home page (root docs.crossplane) to call out the Join Slack link. 
Screenshot 2023-05-15 at 12 54 38 PM

This is only shown on the home page and not in the documentation pages. 
The notification disappears on mobile when the nav changes.

Resolves #201 

Signed-off-by: Pete Lumbis <pete@upbound.io>

### CODE REVIEW

**Comment by jeanduplessis** [comment]
@plumbis the popover doesn't seem to anchor correctly on the deploy preview.

CleanShot 2023-05-15 at 19 05 33@2x

**Comment by plumbis** [comment]
@jeanduplessis this was a PurgeCSS issue. I pushed a commit to fix it.

File: themes/geekboot/layouts/partials/docs-navbar.html [hunk 1]
```diff
@@ -75,9 +75,16 @@
...
 
             <li class="nav-item col-xs p-2">
-              <a class="navbar-link" href="https://slack.crossplane.io" title="Join the Crossplane Slack" target="_blank" rel="noopener">
+              <div id="slack"><a class="navbar-link faa-pulse" href="https://slack.crossplane.io" title="Join the Crossplane Slack" target="_blank" rel="noopener">
                 {{ partialCached "icons/slack.svg" . }} <span class="icon-label">Crossplane Slack</span>
               </a>
+            </div>
+            {{ if .IsHome }}
+              <div id="tooltip" class="" role="tooltip">
+                Join the Crossplane Slack
```
Comment (jeanduplessis) [inline themes/geekboot/layouts/partials/docs-navbar.html:84]: Could we consider making the text clickable? When I view it, I want to click on it. It feels like a call to action, but its weird that I can't click on it.

**Review by jeanduplessis — CHANGES_REQUESTED** [review]

File: themes/geekboot/layouts/partials/docs-navbar.html (hunk 1 above)
Comment (plumbis) [reply themes/geekboot/layouts/partials/docs-navbar.html:85]: Ask and you shall receive.

**Review by jeanduplessis — APPROVED** [review]

**Pull request merged**